sdk/src/types/block/address/ed25519.rs

```diff
@@ -24,6 +24,11 @@ impl Ed25519Address {
     pub fn new(address: [u8; Self::LENGTH]) -> Self {
         Self::from(address)
     }
+
+    #[allow(dead_code)]
```

Comment:
Is this really needed since we actually use it?

Reply:
features